example-app/src/style.css

```
@@ -1,119 +1,124 @@
 
-pre#plugin-output {
+#plugin-output {
   margin: 0;
-  padding: 1.2rem;
-  background: rgba(15, 23, 42, 0.6);
-  border-radius: 12px;
-  font-family: 'JetBrains Mono', 'Fira Code', ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, 'Liberation Mono', 'Courier New', monospace;
-  max-height: 280px;
-  overflow-y: auto;
   white-space: pre-wrap;
   word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)